src/FormatFs.fs

```diff
@@ -6,7 +6,7 @@ open Fantomas
 open FSharp.Compiler.SourceCodeServices
 
 let formatAst ast =
-    let cfg = { FormatConfig.FormatConfig.Default with StrictMode = true; PageWidth = 120 } // do not format comments
```

Comment:
The default value of PageWidth is already 120 for some time now.
The setting has also been renamed to `MaxLineLength` closer match the .editorconfig counterpart.